syntax/div.php

```diff
@@ -8,14 +8,6 @@
  * @author     Esther Brunner <esther@kaffeehaus.ch>
  */
 
-// must be run within DokuWiki
-if(!defined('DOKU_INC')) die();
-
-// global used to indicate that the localised folder link title tooltips 
-// strings have been written out
-global $plugin_folded_strings_set;
-if (!isset($plugin_folded_string_set)) $plugin_folded_string_set = false;
-
```

Comment:
These global are something from the past? They are not used in the code at least.

Reply:
I wondered like you where it's coming from. But I did not check the code history. Just checked that it's not used and quickly tested in my DokuWiki installation. Seemed to work OK so I removed the lines.